src/main/java/net/seesharpsoft/intellij/plugins/csv/editor/table/swing/CsvTableEditorSwing.java

```diff
@@ -255,6 +266,9 @@ protected void afterTableComponentUpdate(Object[][] values) {
         try {
             DefaultTableModel tableModel = this.getTableModel();
             tableModel.setColumnIdentifiers(generateColumnIdentifiers(values, tableModel.getColumnCount()));
+            int baseFontSize = getGlobalFontSize();
+            setFontSize(baseFontSize);
+            baseFontHeight = getFontHeight();
```

Comment:
why creating a variable `baseFontHeight` instead of a `getBaseFontHeight` function?

Reply:
becuase to get font height, we must have the font.

When the table font size is zoomed ,  the font is changed too. so we must use a property to save the original font height  or  the original font.